storage/feed.go

```diff
@@ -283,7 +284,11 @@ func (s *Storage) FeedByID(userID, feedID int64) (*model.Feed, error) {
 			f.category_id,
 			c.title as category_title,
 			fi.icon_id,
-			u.timezone
+			u.timezone,
+			(SELECT count(*) FROM entries 
+				WHERE entries.feed_id=f.id AND 
+				(entries.published_at Between (now() - interval '7 days') and now()) 
+				) as week_count
```

Comment:
Why put this logic in this function? It's seems to be required only at one place. I would move this part to a separate function.